tutorials/tutorial5/tutorial_5.rb

```diff
@@ -0,0 +1,25 @@
+require "date"
+current_day = Date.today
+day_name = current_day.strftime("%A")
+
```

Comment:
Instead using line no 5 to 25, u can use loop condition for this case. U can reference from below codes.

`6.times do |i|
  last_days = current_day - i
  last_days_name = last_days.strftime("%A")
  puts "Today is  #{last_days} ( #{last_days_name} )" if i == 0
  i == 1? (puts "1. Yesterday is  #{last_days} ( #{last_days_name} )") :
          (puts "#{i}. Last #{i} day is #{last_days} ( #{last_days_name} )") unless i == 0
end`

Reply:
Thanks for guiding me, bro @HtetTunLu .I updated according to your instruction on "tutorial5.rb" with this commit [b713899](https://github.com/maythuzarlinn/Ruby-Training/pull/5/commits/b713899ea29394fa9e78a43c7ed37c35edcd0e39)